### PR TITLE
fix(deps): removes uuid lib and remove crypto dependency

### PR DESCRIPTION
**Pull request by andrewxhill**
required by https://github.com/textileio/js-threads-client/issues/209

fixes https://github.com/textileio/js-threads-client/issues/210

### CODE REVIEW

**Comment by andrewxhill** [comment]
otherwise you need something like, https://github.com/LinusU/react-native-get-random-values to run on RN

**Comment by carsonfarmer** [comment]
We should actually just use this opportunity to switch to ulid, though not sure if that has an annoying crypto requirement as well?

File: package.json [hunk 1]
```diff
@@ -68,7 +69,7 @@
     "@types/mocha": "^7.0.1",
     "@types/multibase": "^0.6.0",
     "@types/next-tick": "^1.0.0",
-    "@types/uuid": "^7.0.2",
+    "@types/uuid": "^3.4.0",
```
Comment (carsonfarmer) [inline package.json:72]: Looks like this can be removed as well right?

File: src/index.spec.ts [hunk 2]
```diff
@@ -93,57 +93,49 @@ describe('Client', function () {
   })
   describe('.create', () => {
     it('response should contain a JSON parsable instancesList', async () => {
-      const entities = await client.create<Person>(dbID, 'Person', [createPerson()])
-      expect(entities.length).to.equal(1)
+      const instances = await client.create<Person>(dbID, 'Person', [createPerson()])
```
Comment (carsonfarmer) [inline src/index.spec.ts:96]: 👍

File: src/index.spec.ts [hunk 3]
```diff
@@ -93,57 +93,49 @@ describe('Client', function () {
...
-      const q: QueryJSON = {
-        ands: [
-          {
-            fieldPath: 'firstName',
-            operation: ComparisonJSON.Eq,
-            value: { string: frank.firstName },
-          },
-        ],
-      }
+      const q = new Where('firstName').eq(frank.firstName)
```
Comment (carsonfarmer) [inline src/index.spec.ts:138]: _much_ nicer

**Pull request merged**